### DISCUSSION TITLE
Ping response is {OK} but it is logged as a simple Log instead of an OK status

### DISCUSSION THREAD

**Discussion by lerufino** [healthchecks/healthchecks#1079] [Q&A]
I'm using an application called [Backrest ](https://github.com/garethgeorge/backrest) on Windows to send pings to a Check. I have tried two methods, Backrest's direct integration to Healthcheck and Backrest calling a .bat file with the ping. Neither worked as expected. Despite receiving an OK status in Backrest's log, it is unable to make a new check alive. And when the check is already alive (for example, if I externally used the .bat to start it manually) it is unable to keep it alive. In Healthcheck's logs it show that I have been pokint them, but it register as a Log, and not an OK status.

## **The Backrest direct ping register a log like this:**
```
[tasklog] 23:10:01.750Z	info	run hook healthchecks hook plan/plan1/hook/1	Sending healthchecks message to https://hc-ping.com/5fccf55c-b105-4b09-b1fc-ddc7fba3157b

[tasklog] 23:10:01.751Z	debug	run hook healthchecks hook plan/plan1/hook/1	Sending healthchecks message	{"payload": "2024-10-26T23:10:01-03:00"}

[tasklog] 23:10:02.644Z	debug	run hook healthchecks hook plan/plan1/hook/1	Healthchecks response	{"body": "OK"}

[tasklog] 23:10:02.644Z	info	run hook healthchecks hook plan/plan1/hook/1	task finished	{"duration": 0.8939521}
```

## **While Healthcheck's log shows the following:**
![image](https://github.com/user-attachments/assets/23360cad-d456-4f62-8f55-cee484091a16)

> *This check is Ready to receive pings, but never logged one. Every ping is just a Log, incapable of keeping the Check alive.*

![image](https://github.com/user-attachments/assets/b802525a-f4c8-4577-a913-5416be0b9b6c)
> *Is there something wrong with the fact that Backrest is pinging the /Log endpoint? Should it be different?*

____
Since I have not full control over what the Backrest does when directly pinging can there be a parameter being sent that tells Healthcheck not to register it as ok? Why does it not register the received ping as an OK status?

**Comment by cuu508** [comment]
When you hook up Healthchecks.io with Backrest, do you specify an UUID or a full URL? If it's an URL, what URL did you use? If the URL ends with `/log`, remove that part.

The log endpoint is used for sending logging information to Healthchecks.io without signaling success or failure. https://healthchecks.io/docs/http_api/#log-uuid

**Reply by lerufino** [reply]
I thought it would be something like that, that the log endpoint was not really meant to ping. If that is the case, that Backrest is opting to use /log i might want to have some more feedback from them.

The config I set up in Backlog looks like this:

```{ "conditions": ["CONDITION_SNAPSHOT_END"], "actionHealthchecks": { "webhookUrl": "https://hc-ping.com/API_KEY", "template": "{{ .Summary }}" } }```

I doesn't seems to be using /log, but maybe it does. I'm trying to reach Backrest for help too.
